### PR TITLE
repo、category page progressive load app data

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/Rectangle/AppImages/index.jsx [hunk 1]
```diff
@@ -8,11 +8,13 @@ import styles from './index.scss';
 
 export default class AppImages extends Component {
   static propTypes = {
-    apps: PropTypes.array
+    apps: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    total: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
```
Comment (iwisunny) [inline src/components/Rectangle/AppImages/index.jsx]: why add `total` prop? Does this prop can be produced by `apps`?

File: src/components/Rectangle/index.jsx [hunk 2]
```diff
@@ -10,12 +10,13 @@ export default class Rectangle extends PureComponent {
   static propTypes = {
     title: PropTypes.string,
     description: PropTypes.string,
-    apps: PropTypes.array,
+    apps: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    total: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```
Comment (iwisunny) [inline src/components/Rectangle/index.jsx]: `total` can be restricted to `number` type

File: src/components/Rectangle/AppImages/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/components/Rectangle/AppImages/index.jsx]: total is total_count of api result , apps is some ahead apps  of category apps.

File: src/components/Rectangle/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/components/Rectangle/index.jsx]: not load apps data, number defalut  '', not 0.

**Pull request merged**